src/libraries/Common/src/System/Security/Cryptography/ECDsaCng.ImportExport.cs

```diff
@@ -105,18 +113,38 @@ public override ECParameters ExportParameters(bool includePrivateParameters)
         {
             ECParameters ecparams = default;
 
+            const string TemporaryExportPassword = "DotnetExportPhrase";
```

Comment:
Are we going to have a problem with CredScan after we merge this? We should try pushing this change to the internal mirror and make sure it doesn't trip CredScan before we merge this.

Reply:
I was back-channeling with @GrabYourPitchforks on this - we (well, Levi) is going to run it - especially since this might be a thing we try to fix for 9.0.